{
	"schema_version": "1.6.0",
	"id": "CVE-2023-4459",
	"modified": "2024-09-18T03:25:30.592356Z",
	"published": "2023-08-21T19:15:09Z",
	"related": [
		"RHSA-2024:0412",
		"RHSA-2024:1250",
		"RHSA-2024:1306",
		"RHSA-2024:1367",
		"RHSA-2024:1382",
		"RHSA-2024:2006",
		"RHSA-2024:2008",
		"SUSE-SU-2023:3599-1",
		"SUSE-SU-2023:3599-2",
		"SUSE-SU-2023:3600-1",
		"SUSE-SU-2023:3600-2",
		"SUSE-SU-2023:3601-1",
		"SUSE-SU-2023:3656-1",
		"SUSE-SU-2023:3680-1",
		"SUSE-SU-2023:3681-1",
		"SUSE-SU-2023:3682-1",
		"SUSE-SU-2023:3684-1",
		"SUSE-SU-2023:3687-1",
		"SUSE-SU-2023:3705-1",
		"SUSE-SU-2023:3785-1",
		"SUSE-SU-2023:3964-1",
		"SUSE-SU-2023:3969-1",
		"SUSE-SU-2023:3971-1",
		"SUSE-SU-2023:3988-1",
		"SUSE-SU-2023:4028-1",
		"UBUNTU-CVE-2023-4459",
		"USN-6388-1"
	],
	"details": "A NULL pointer dereference flaw was found in vmxnet3_rq_cleanup in drivers/net/vmxnet3/vmxnet3_drv.c in the networking sub-component in vmxnet3 in the Linux Kernel. This issue may allow a local attacker with normal user privilege to cause a denial of service due to a missing sanity check during cleanup.",
	"severity": [
		{
			"type": "CVSS_V3",
			"score": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H"
		}
	],
	"affected": [
		{
			"package": {
				"ecosystem": "Debian:11",
				"name": "linux",
				"purl": "pkg:deb/debian/linux?arch=source"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "5.10.120-1"
						}
					]
				}
			],
			"versions": [
				"5.10.103-1",
				"5.10.103-1~bpo10+1",
				"5.10.106-1",
				"5.10.113-1",
				"5.10.120-1~bpo10+1",
				"5.10.46-4",
				"5.10.46-5",
				"5.10.70-1",
				"5.10.70-1~bpo10+1",
				"5.10.84-1",
				"5.10.92-1",
				"5.10.92-1~bpo10+1",
				"5.10.92-2"
			],
			"ecosystem_specific": {
				"urgency": "not yet assigned"
			},
			"database_specific": {
				"source": "https://storage.googleapis.com/cve-osv-conversion/osv-output/CVE-2023-4459.json"
			}
		},
		{
			"package": {
				"ecosystem": "Debian:12",
				"name": "linux",
				"purl": "pkg:deb/debian/linux?arch=source"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "5.17.11-1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"urgency": "not yet assigned"
			},
			"database_specific": {
				"source": "https://storage.googleapis.com/cve-osv-conversion/osv-output/CVE-2023-4459.json"
			}
		},
		{
			"package": {
				"ecosystem": "Debian:13",
				"name": "linux",
				"purl": "pkg:deb/debian/linux?arch=source"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "5.17.11-1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"urgency": "not yet assigned"
			},
			"database_specific": {
				"source": "https://storage.googleapis.com/cve-osv-conversion/osv-output/CVE-2023-4459.json"
			}
		}
	],
	"references": [
		{
			"type": "ADVISORY",
			"url": "https://access.redhat.com/errata/RHSA-2024:0412"
		},
		{
			"type": "ADVISORY",
			"url": "https://access.redhat.com/errata/RHSA-2024:1250"
		},
		{
			"type": "ADVISORY",
			"url": "https://access.redhat.com/errata/RHSA-2024:1306"
		},
		{
			"type": "ADVISORY",
			"url": "https://access.redhat.com/errata/RHSA-2024:1367"
		},
		{
			"type": "ADVISORY",
			"url": "https://access.redhat.com/errata/RHSA-2024:1382"
		},
		{
			"type": "ADVISORY",
			"url": "https://access.redhat.com/errata/RHSA-2024:2006"
		},
		{
			"type": "ADVISORY",
			"url": "https://access.redhat.com/errata/RHSA-2024:2008"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=2219268"
		},
		{
			"type": "FIX",
			"url": "https://github.com/torvalds/linux/commit/edf410cb74dc612fd47ef5be319c5a0bcd6e6ccd"
		},
		{
			"type": "WEB",
			"url": "https://access.redhat.com/security/cve/CVE-2023-4459"
		},
		{
			"type": "ADVISORY",
			"url": "https://security-tracker.debian.org/tracker/CVE-2023-4459"
		}
	]
}
